### DISCUSSION TITLE
How do you load your own color theme in customtkinter?

### DISCUSSION THREAD

**Discussion by franjjapa6** [TomSchimansky/CustomTkinter#1060] [Q&A]
Hi, so I just wrote my own customtkinter color theme and I'm having trouble loading it in. Any tips or suggestions?

**Comment by avalon60** [comment]
Hi @franjjapa6, just use the pathname to the JSON.
Example:
`customtkinter.set_default_color_theme(str(themes_location / f'{self.app_theme}.json'))`
